pandas/core/config_init.py

```diff
@@ -362,7 +362,7 @@ def table_schema_cb(key):
             from IPython.core.formatters import BaseFormatter
 
             class TableSchemaFormatter(BaseFormatter):
-                print_method = '_repr_table_schema_'
```

Comment:
@TomAugspurger pls move all of the table_schema code out of config_init into somewhere else; the cb should be really short/simple and not include all of this code